wfs/wfs_egib.py

```diff
@@ -92,22 +91,23 @@ def save_gml(self, folder, url, teryt):
         if name_error == "brak":
             url_main = url.split('?')[0]
 
-            name_error_lista = []
             name_error_lista_brak = []
 
+            name_error_lista = []
             for layer in name_layers:
-
                 if prefix == 'ewns':
-                    url_gml = url_main + f"?service=WFS&request=GetFeature&version=2.0.0&typeNames={layer}&namespaces=xmlns(ewns,http://xsd.geoportal2.pl/ewns)"
+                    url_gml = f"{url_main}?service=WFS&request=GetFeature&version=2.0.0&typeNames={layer}&namespaces=xmlns(ewns,http://xsd.geoportal2.pl/ewns)"
                 elif prefix == 'ms':
-                    url_gml = url_main + f"?service=WFS&request=GetFeature&version=1.0.0&typeNames={layer}&namespaces=xmlns(ms,http://mapserver.gis.umn.edu/mapserver)"
+                    url_gml = f"{url_main}?service=WFS&request=GetFeature&version=1.0.0&typeNames={layer}&namespaces=xmlns(ms,http://mapserver.gis.umn.edu/mapserver)"
                 else:
-                    url_gml = url_main + '?request=getFeature&version=2.0.0&service=WFS&typename=' + layer
+                    url_gml = f'{url_main}?request=getFeature&version=2.0.0&service=WFS&typename={layer}'
```

Comment:
Zamień tu proszę pojedynczy cudzysłów na podwójny, może to czasami powodować problemy

Reply:
z tym się nie zgodzę, pep8 mówi że pojedyncze i podwójne cudzysłowy są równorzędne - https://peps.python.org/pep-0008/#string-quotes a poza tym od początku działania na pobieraczu staram się, tam gdzie mogę, używać pojedynczych cudzysłowów

Reply:
dobra to na razie niech będzie tak jak jest